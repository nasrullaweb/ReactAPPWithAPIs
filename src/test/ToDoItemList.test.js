import React from 'react';
import ToDoList from '.././Components/ToDoItemList';
import renderer from 'react-test-renderer';

test('Render todo List', () => {
    const component = renderer.create(
        <ToDoList 
          items={[{text:'first list', id:1}, {text:'Second list', id:2}]}
        />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

  });

  

