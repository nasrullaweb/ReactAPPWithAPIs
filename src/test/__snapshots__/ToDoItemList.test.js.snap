// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render todo List 1`] = `
<ul
  className="theList"
>
  <li>
    first list
    <span
      className="remove"
      onClick={[Function]}
    >
      Remove
    </span>
    <span
      className="update"
      onClick={[Function]}
    >
      EDIT
    </span>
  </li>
  <li>
    Second list
    <span
      className="remove"
      onClick={[Function]}
    >
      Remove
    </span>
    <span
      className="update"
      onClick={[Function]}
    >
      EDIT
    </span>
  </li>
</ul>
`;
